ansible/playbooks/adhoc/hotfix/roles/update_node/tasks/main.yml

```diff
@@ -53,7 +51,7 @@
   service:
     name: atomic-openshift-node
     state: restarted
-  when: "{{ update_needed == True and update_successful == True }}" 
+  when: "{{ update_needed == True and upgrade_successful == True }}"
```

Comment:
Same, change upgrade_successful to something that covers update and downgrade better.